### PR TITLE
koord-descheduler: set default values for evictQPS/evictBurst

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅰ. Describe what this PR does

1. Set evictQPS of MigrationControllerArgs to 10
2. Set evictBurst of MigrationControllerArgs to 1
3. evictQPS support string or float64 type and users expect to write numbers directly.
4. Set MaxMigratingPerNode to 2

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/descheduler/apis/config/v1alpha2/types_pluginargs.go [hunk 1]
```diff
@@ -143,9 +145,9 @@ type MigrationControllerArgs struct {
 	DefaultJobTTL *metav1.Duration `json:"defaultJobTTL,omitempty"`
 
 	// EvictQPS controls the number of evict per second
-	EvictQPS string `json:"evictQPS,omitempty"`
+	EvictQPS *config.Float64OrString `json:"evictQPS,omitempty"`
```
Comment (jasonliu747) [inline pkg/descheduler/apis/config/v1alpha2/types_pluginargs.go:148]: tha's very thoughtful, but is it really necessary?

Comment (eahydra) [reply pkg/descheduler/apis/config/v1alpha2/types_pluginargs.go:148]: Yes..Users naturally think that the evictQPS field needs to be filled with numbers and needs to be of floating point type. In order to be compatible with stock scenarios, it is now necessary to support both floating point and string types

Comment (jasonliu747) [reply pkg/descheduler/apis/config/v1alpha2/types_pluginargs.go:148]: I see, that makes sense to me.

**Review by FillZpp — APPROVED** [review]
/lgtm

**Pull request merged**